pxr/usd/usdLightField/schema.usda

```diff
@@ -0,0 +1,440 @@
+                available it should be preferred."""
+    )
+}
+
+class "LightFieldKernelBaseAPI" (
+    doc = """Defines a base-class type applied schema that all applied schema
+            that provides the a ParticleField kernel will automatically apply.
+            The purpose of this base class is to allow validation to enforce
+            that a kernel definition is present for a ParticleField"""
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "KernelBaseAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+    }
+)
+{
+}
+
+class "LightFieldKernelGaussianEllipsoidAPI" (
+    doc = """Defines the gaussian ellipsoid kernel for a given ParticleField.
+
+    The kernel shape is a spherical region that is reshaped by the associated
+    scale data source, and rotated by the orientation data source.
+
+    The falloff function for this kernel is the gaussian falloff function,
+    where the peak of the falloff function is defined by the opacity data
+    source.
+    """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "KernelGaussianEllipsoidAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+        string extraIncludes = """
+            #include "pxr/usd/usdLightField/kernelBaseAPI.h"
+        """
+        token[] reflectedAPISchemas = [
+            "LightFieldKernelBaseAPI"
+        ]
+    }
+    prepend apiSchemas = [
+        "LightFieldKernelBaseAPI"
+    ]
+)
+{
+}
+
+class "LightFieldKernelGaussianSurfletAPI" (
+    doc = """Defines the gaussian surflet kernel for a given ParticleField.
+
+    The kernal shape is a circular disk centered at the origin, with the normal
+    of the plane being oriented along the positive z-axis.
+
+    The falloff function for this kernel is the gaussian falloff function,
+    where the peak of the falloff function is defined by the opacity data
+    source.
+    """
+    inherits = </APISchemaBase>
+    customData = {
+        string className = "KernelGaussianSurfletAPI"
+        token apiSchemaType = "singleApply"
+        token[] apiSchemaCanOnlyApplyTo = ["ParticleField"]
+        string extraIncludes = """
+            #include "pxr/usd/usdLightField/kernelBaseAPI.h"
+        """
+        token[] reflectedAPISchemas = [
+            "LightFieldKernelBaseAPI"
+        ]
+    }
+    prepend apiSchemas = [
+        "LightFieldKernelBaseAPI"
+    ]
+)
+{
+}
+
+class "LightFieldKernelConstantSurfletAPI" (
+    doc = """Defines the constant surflet kernel for a given ParticleField.
+
+    The kernal shape is a circular disk centered at the origin, with the normal
+    of the plane being oriented along the positive z-axis.
+
+    The falloff function for this kernel is constant and the value is directly
+    defined by the opacity data source."""
```

Comment:
So the gaussian goes to infinity, but eventually gets insignificantly small.  Theoretically, a circular disk with a constant function for support is a plane?  Or do you mean a box function, or a triangle function?